docs/test/PerfTestPlan-Scalability.adoc

```diff
@@ -74,25 +72,36 @@ to upgrade your google cloud SDK
 .. Go to Arktos source code folder: `cd ~/go/src/arktos`
 .. Build Arktos: `make clean; make quick-release`
 . Start kubemark:
-.. Set env: `export MASTER_ROOT_DISK_SIZE=100GB MASTER_DISK_SIZE=200GB KUBE_GCE_ZONE=us-west2-b MASTER_SIZE=n1-highmem-32 NODE_SIZE=n1-highmem-8 NUM_NODES=8 NODE_DISK_SIZE=200GB KUBE_GCE_NETWORK=kubemark-500 GOPATH=$HOME/go KUBE_GCE_ENABLE_IP_ALIASES=true KUBE_GCE_PRIVATE_CLUSTER=true CREATE_CUSTOM_NETWORK=true KUBE_GCE_INSTANCE_PREFIX=kubemark-500 APISERVERS_EXTRA_NUM=0 PROMETHEUS_SCRAPE_ETCD=true ENABLE_PROMETHEUS_SERVER=true`
+.. Set env: `export MASTER_ROOT_DISK_SIZE=100GB MASTER_DISK_SIZE=200GB KUBE_GCE_ZONE=us-west2-b MASTER_SIZE=n1-highmem-32 NODE_SIZE=n1-highmem-8 NUM_NODES=8 NODE_DISK_SIZE=200GB KUBE_GCE_NETWORK=kubemark-500 GOPATH=$HOME/go KUBE_GCE_ENABLE_IP_ALIASES=true KUBE_GCE_PRIVATE_CLUSTER=true CREATE_CUSTOM_NETWORK=true KUBE_GCE_INSTANCE_PREFIX=kubemark-500 APISERVERS_EXTRA_NUM=0 WORKLOADCONTROLLER_EXTRA_NUM=0`
 .. Start admin cluster: `./cluster/kube-up.sh`
 .. Start kubemark cluster: `./test/kubemark/start-kubemark.sh`
 .. Edit hollow node # (minimal 100 to run perf test) by updating "replicas: 10": `kubectl edit replicationcontroller hollow-node -n kubemark`
 .. Wait for all hollow-nodes are ready (expecting replicas+2): `./cluster/kubectl.sh --kubeconfig=<path to kubeconfig.kubemark> get nodes | wc -l`
 . Start perf test:
 .. Go to perf-test source folder: `cd ~/go/src/k8s.io/perf-tests/clusterloader2/`
-.. Set env: `export PROMETHEUS_SCRAPE_ETCD=true ENABLE_PROMETHEUS_SERVER=true APISERVERS_EXTRA_NUM=0 GOPATH=$HOME/go`
+.. Set env: `export APISERVERS_EXTRA_NUM=0 GOPATH=$HOME/go`
 .. Run perf test: `nohup ./run-e2e.sh --nodes=500 --provider=kubemark --kubeconfig=<path to kubeconfig.kubemark> --report-dir=<folder to save report> --testconfig=testing/load/config.yaml --testconfig=testing/density/config.yaml --testoverrides=./testing/experiments/enable_prometheus_api_responsiveness.yaml --testoverrides=./testing/experiments/use_simple_latency_query.yaml`
 . Shut down cluster
 .. Make sure environment variables set up above still exist as GCP console consistently disconnect
 .. In arktos folder, stop kubemark: `cd ~/go/src/arktos; ./test/kubemark/stop-kubemark.sh`
 .. Shut down cluster: `./cluster/kube-down.sh`
 
 == Metrics
+[width=90%,options="header"]
+|=========
+|Metrics|Description
+|APIResponsiveness|Summary for latency and number for server api calls
+|PodStartupLatency|Verifies if [pod startup SLO] is satisfied
+|SaturationPodStartupLatency|Verifies if [pod startup SLO] is satisfied with limit: CpuRequest: 1m/MemoryRequest: 10M
+|ResourceUsageSummary|The resource usage per component
+|SchedulingThroughput|Scheduling throughput
```

Comment:
Perhaps change description to "Pod successfully scheduled per second"? The more beginner can understand the better